### PR TITLE
feat: pdf-view

### PR DESCRIPTION
**Pull request by grigala**
Ported from: https://github.com/madmas/nativescript-pdf-view

@madmas could you follow up and maybe do a review?

- [x] Angular demo is missing so far.

### CODE REVIEW

**Comment by grigala** [comment]
@cla-bot check

File: packages/pdf-view/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@nativescript/pdf-view",
+  "version": "1.0.0",
+  "description": "A NativeScript plugin to display PDF files on iOS and Android",
+  "main": "pdf-view",
+  "typings": "pdf-view.d.ts",
```
Comment (NathanWalker) [inline packages/pdf-view/package.json]: I would probably stick with `index.d.ts` and `index` for main - can just:
`pdf-view.android.ts` > `index.android.ts`
`pdf-view.ios.ts` > `index.ios.ts`
`pdf-view.common.ts` > `common.ts`
Then can just import the common imports is all. That follows more of a streamlined pattern 👍

**Comment by NathanWalker** [comment]
@grigala this is great! thank you for kicking this off.

**Comment by NathanWalker** [comment]
We will likely rename this to just `pdf` as we will likely in the future work in other pdf features into this, for example also allowing PDFKit usage on iOS (and similar on android):
https://stackblitz.com/edit/nativescript-pdfview-via-ios-pdfkit?file=src%2Fapp%2Fnative-pdfview%2Fnative-pdfview.ts

File: packages/pdf-view/package.json (hunk 1 above)
Comment (grigala) [reply packages/pdf-view/package.json]: Renamed files and changed the module name to `pdf` as you suggested.

**Comment by madmas** [comment]
@NathanWalker we completed the work on this from our perspective :) Demos are there and running for both Angular and Plain for iOS and Android

**Review by NathanWalker — APPROVED** [review]

**Comment by NathanWalker** [comment]
This is amazing everyone. We’ll try to publish this before end of week.

**Comment by NathanWalker** [comment]
@nativescript/pdf@1.0.0 published now. 🤗 

We will likely add more features to this over time as mentioned, like including ability to create pdf's as well as convenience to go beyond just a viewer.

**Pull request merged**